ballerina/annotation.bal

```diff
@@ -44,3 +44,9 @@ public type SubscriberServiceConfiguration record {|
 
 # WebSub Subscriber Configuration for the service, indicating subscription related parameters.
 public annotation SubscriberServiceConfiguration SubscriberServiceConfig on service;
+
+public type MetaInformation record {|
+    byte[] servicePath;
+|};
+
+public annotation MetaInformation MetaInfo on service;
```

Comment:
Is this public annotation? If so we need docs

Reply:
Fixed with [bdfe5bb4dc40458438ae4849d58d8f61bb6840e3](https://github.com/ballerina-platform/module-ballerina-websub/pull/510/commits/bdfe5bb4dc40458438ae4849d58d8f61bb6840e3)